docs/about.html

```diff
@@ -122,11 +122,13 @@ <h2 class="display-4">RaiderPlanner:<br>The Way Ahead</h2>
     </section>
 
     <!-- Footer -->
-    <footer class="py-3 bg-dark">
+    <footer class="py-1 bg-dark">
       <div class="footer">
         <p> Would you like to contribute to this project? Click <a href="contribute.html">HERE</a> to find out more information </p>
-        <p class="m-0 text-center text-white">Copyright &copy; RaiderPlanner 2017</p>
+        <p class="m-0 text-center text-white">Copyright &copy; <a class="nav-link" href="../AUTHORS.txt">RaiderPlanner Contributors</a></p>
```

Comment:
This should still have the year, 2017. Also, note that the file in the repository is simply called AUTHORS, with no extension. Clicking on the link results in the file not being found.